.github/dependabot.yml

```diff
@@ -4,6 +4,8 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+    commit-message:
+      prefix: ci
```

Comment:
Nice, I didn't know you could influence the commit message of dependabot updates.